ui/Makefile

```
@@ -1,20 +1,16 @@
 UI_BASE_URL ?= https://localhost:8000
-YARN_NETWORK_TIMEOUT ?= 60000
+UI_PKG_INSTALL_NETWORK_TIMEOUT ?= 60000
 
 .PHONY: all
 all: deps lint test test-component build
 
 SOURCES := $(shell find apps -type d \( -name node_modules \) -prune -o -print)
 
-# in monorepo yarn.lock might remain unchanged when new NPM packages are created,
-# tracking package.json files of monorepo packages is the easiest way to discover such situations
-PACKAGE_JSON_FILES := $(shell find . -type d \( -name node_modules \) -prune -false -o -name package.json)
-
 ROX_PRODUCT_BRANDING ?= $(shell $(MAKE) --quiet --no-print-directory -C .. product-branding)
 export REACT_APP_ROX_PRODUCT_BRANDING := $(ROX_PRODUCT_BRANDING)
 
-deps: apps/platform/yarn.lock $(PACKAGE_JSON_FILES)
-	yarn install --cwd=apps/platform --frozen-lockfile --prefer-offline --network-timeout=$(YARN_NETWORK_TIMEOUT)
+deps: apps/platform/package.json apps/platform/package-lock.json
+	npm ci --prefix apps/platform --prefer-offline --fetch-timeout=$(UI_PKG_INSTALL_NETWORK_TIMEOUT)
```

[optional] I hope it's not too controversial to add `--no-fund` to reduce output.
```suggestion
	npm ci --prefix apps/platform --prefer-offline --fetch-timeout=$(UI_PKG_INSTALL_NETWORK_TIMEOUT) --no-fund
```

---

Works for me, I don't think CI is going to send funding to anyone 😄